__tests__/pages/ab-testing/demographic-data-survey.test.js

```diff
@@ -131,89 +184,52 @@ describe("Questionnaire démographique", () => {
 
       expect(
         checkIsFormCompleted(
-          genderValues,
+          genderValuesCompleted,
           ageValues,
           undefined,
           situationValues,
           entourageValues
         )
       ).toBeFalsy()
     })
-    test("Genre + Age + CP + situation + entourage complétés => true", () => {
-      const genderValues = [
-        {
-          id: "femme",
-          text: "Femme",
-          isChecked: true,
-        },
-        {
-          id: "homme",
-          text: "Homme",
-          isChecked: false,
-        },
-      ]
 
-      const ageValues = [
-        {
-          id: "-18ans",
-          text: "-18 ans",
-          isChecked: true,
-        },
-        {
-          id: "18-25ans",
-          text: "18 - 25 ans",
-          isChecked: false,
-        },
-      ]
-
-      const jobValues = { code: "10", libelle: "job" }
-
-      const situationValues = [
-        {
-          id: "attendez1Enfant",
-          text: "Vous attendez un enfant",
-          isChecked: true,
-        },
-        {
-          id: "avezEnfantDeMoinsDe2ans",
-          text: "Vous avez un enfant de moins de 2 ans",
-          isChecked: false,
-        },
-      ]
-
-      const entourageValues = [
-        {
-          id: "oui",
-          text: "Oui",
-          isChecked: true,
-        },
-        {
-          id: "non",
-          text: "Non",
-          isChecked: false,
-        },
-      ]
-
-      const residenceValue = {
-        zipcode: "49000",
-        city: "Écouflant",
-        departmentName: "Maine-et-Loire",
-        departmentNumber: "49",
-        region: "Pays de la Loire",
-        label1Bold: "Écouflant",
-        label2: "49, Maine-et-Loire, Pays de la Loire",
-      }
+    test("Should return true when all form is completed and zip code is an object", () => {
+      expect(
+        checkIsFormCompleted(
+          genderValuesCompleted,
+          ageValuesCompleted,
+          jobValuesCompleted,
+          residenceValueCompleted,
+          situationValuesCompleted,
+          entourageValuesCompleted
+        )
+      ).toBeTruthy()
+    })
 
+    test("Should return true when all form is completed and zip code is true", () => {
```

Comment:
`zip code` is true / false, ça veut dire est-ce qu'il est présent ou valide ? je suis pas sûre de comprendre exactement

Reply:
à la base, je renvoyais un objet, et finalement je renvoi un true/false, mais j'ai gardé le fait de pouvoir renvoyé un objet qd même. Il faut juste que ca ne soit pas `undefined`